src/www/status_dhcpv6_leases.php

```diff
@@ -50,11 +50,11 @@ function adjust_gmt($dt)
 
     $timezone = $config['system']['timezone'];
     $ts = strtotime($dt . " GMT");
-    if ($dhcpv6leaseinlocaltime == "yes") {
+    if ($dhcpv6leaseinlocaltime != "yes") {
```

Comment:
I don't know, this looked more correct before: assume time is GMT and if not add the offset (it can be negative) unless we assume the time is never GMT from dhcp in the first place, but then the help text is wrong...

Reply:
leasetime in dhcp will always be local time surely?

Reply:
dunno. some part of the code is definitely still lying to us oO